Raven.Database/Smuggler/SmugglerDatabaseApi.cs

```diff
@@ -61,7 +67,7 @@ public override async Task Between(SmugglerBetweenOptions<RavenConnectionStringO
                         From = exportOperations,
                         To = importOperations,
                         IncrementalKey = betweenOptions.IncrementalKey
-                    }, Options, exportBulkOperation)
+                    }, Options, null)
```

Comment:
It looks like SmugglerBetweenOperations does not require the ExportBulkInsert in any of the cases now. It can be removed from arguments?